src/wasm/kotlin/BUILD

```diff
@@ -3,14 +3,14 @@ load("//tools/build_defs/kotlin:rules.bzl", "kt_jvm_library")
 
 licenses(["notice"])
 
-kt_jvm_library(
+alias(
```

Comment:
Can you add a comment saying what the purpose of these aliases is?

(Probably also worth adding a TODO -- I don't think we want them long term)

Reply:
I'm not sure what the purpose is. I'll leave a TODO

Reply:
I guess a follow up question is: if you delete these aliases, what happens?

I suspect they're here to keep old references working. But we really should just update those references (follow up PR would be fine)